sdk/keyvault/azure-security-keyvault-secrets/README.md

```diff
@@ -16,7 +16,7 @@ Maven dependency for the Azure Key Vault Secrets client library. Add it to your
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-secrets</artifactId>
-    <version>4.3.4</version>
+    <version>4.4.0-beta.3</version>
```

Comment:
README versions should point to stable.

Reply:
I think the ones that will be included with a beta release's tag should point to the beta version, after releasing said version I'll update the READMEs to point to stable.